### PR TITLE
masvis: Add version 1.2

### PR DESCRIPTION
**Pull request by davidxuang**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
`persist` is not required because config is stored at `$Env:AppData\TolvanData\MasVis\$version`.
(This is just a reminder, no need to change anything.)

**Review by issaclin32 — APPROVED** [review]

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/masvis.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "1.2",
+    "description": "Stereo audio analyzer for CD-ripping",
+    "homepage": "https://www.lts.a.se/lts/masvis",
+    "license": "Freeware",
+    "url": "http://www.lts.a.se/pub/masvis/MasVis.exe",
```
[the remark this answers is not in the document]
Comment (issaclin32) [reply bucket/masvis.json:6]: It is.
```
pelook MasVis.exe
loaded "MasVis.exe" / 2158592 (0x20F000) bytes
signature/type:       PE32 EXE image for i386
image checksum:       0x00000000 (calc=0x00212124)
machine:              0x014C (i386)
subsystem:            2 (Windows GUI)
minimum os:           5.0 (Win2k)
linkver:              Borland TLINK/ILINK 2.25
timestamp:            09/09/2010 07:56:28pm (0x4C893BEC)
```

[the remark this answers is not in the document]
Comment (davidxuang) [reply bucket/masvis.json:6]: Yep

图片

[the remark this answers is not in the document]
Comment (rashil2000) [reply bucket/masvis.json:6]: Okay 👍

File: bucket/masvis.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "1.2",
+    "description": "Stereo audio analyzer for CD-ripping",
+    "homepage": "https://www.lts.a.se/lts/masvis",
+    "license": "Freeware",
```
[the remark this answers is not in the document]
Comment (davidxuang) [reply bucket/masvis.json:5]: https://www-lts-a-se.translate.goog/lts/masvis?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=zh-CN

> MasVis is freeware and runs on Microsoft Windows.

**Pull request merged**